phys/module_fr_fire_driver.F

```diff
@@ -118,7 +127,68 @@ subroutine fire_driver_em ( grid , config_flags                    &
     itimestep=grid%itimestep
     restart=config_flags%restart
 
+    ! time - assume dt does not change
+    dt = grid%dt
+    time_start = itimestep * dt
```

Comment:
@janmandel 
Jan,
What happens with a really small dt? Seems like time start could be zero for a while.

Reply:
Dave,
this is fine, dt and time_start are both REAL, so from starting form itimestep=1, we have also time_start>0. These lines were there before, they were just moved up because time_start is needed earlier now.
Jan